pkgs/python-wrapped/default.nix

```
@@ -0,0 +1,32 @@
+{ pkgs, python, python-ld-library-path }:
+pkgs.buildGoModule rec {
+  pname = "python-wrapped";
+  version = "0.1.0";
+
+  src = ./.;
+
+  ldflags = [
+    "-X main.ReplitPythonLdLibraryPath=${python-ld-library-path}"
+    "-X main.PythonExePath=${python}/bin/python"
+  ];
+
+  vendorHash = null;
+
+  meta = with pkgs.lib; {
+    description = ''
+      A replit proxy to Python which interpolates some critical bits
+      of the environment at runtime
+    '';
+    homepage = "https://replit.com";
+    license = licenses.mit;
+    mainProgram = "python-wrapped";
```

```suggestion
    mainProgram = "python";
```

Pretty sure this should be python, you aren't making a binary available in $out/bin called "python-wrapper".

---

Let me give it another shot -- I'm intentionally offering the binary as `.python-wrapped` with three symlinks, as a sign to users digging in that we're engaging in some shenanigans.

---

mainProgram is only used by `nix run` and it just looks in $out/bin for the name to run. If it's not there, it will probably be messed up. That said, it's not likely someone is going to be nix running this so it doesn't really matter.

---

aah, I thought it was the other way around. OK, I'll make sure it's set.